app/src/main/java/utils/DateManipulation.java

```diff
@@ -27,13 +26,35 @@ public static Boolean isTomorrow(Calendar cal){
         return cal.get(Calendar.DAY_OF_MONTH) == tomorrow.get(Calendar.DAY_OF_MONTH);
     }
 
+    public static Boolean isAfterTomorrow(Calendar cal){
+        Calendar tomorrow = Calendar.getInstance();
+        tomorrow.add(Calendar.DAY_OF_MONTH,2);
+        return cal.get(Calendar.DAY_OF_MONTH) >= tomorrow.get(Calendar.DAY_OF_MONTH);
+    }
```

Comment:
This method isn't working properly because if t's the 28 of the month and we add 4 days, we'll be the 2 of the next month.
So the comparison will be 2 (November) >= 29 (December) is erroned and will return false.

You shoud compare by getting the calendars in milliseconds : 
`return cal.getTimeInMillis() >= tomorrow.getTimeInMillis()`